packages/core/test/io/node-io.test.ts

```diff
@@ -2,11 +2,22 @@ import test from 'tape';
 import { environment, Environment } from '../../../test-utils';
 import { NodeIO } from '@gltf-transform/core';
 
-let fs, glob, path;
+let fs, glob, path, server, ip;
 if (environment === Environment.NODE) {
 	fs = require('fs');
 	glob = require('glob');
 	path = require('path');
+	ip = require("ip").address() + ':' + 8989;
+	const http = require('http');
+
+	server = http.createServer((req, res) => {
+		const filePath = path.join(path.join(__dirname, '../in'), req.url);
+		const file = fs.readFileSync(filePath, 'utf-8');
+		res.write(file);
+		res.end();
+	});
+	
+	server.listen(8989);
 }
```

Comment:
Running tests locally I'm seeing the following error:

```
error: request to http://192.168.1.124:8989/Box_glTF-Binary/Box.glb failed, reason: connect ECONNREFUSED 192.168.1.124:8989
(node:69294) UnhandledPromiseRejectionWarning: FetchError: request to http://192.168.1.124:8989/BoxVertexColors.glb failed, reason: connect ECONNREFUSED 192.168.1.124:8989
    at ClientRequest.<anonymous> (/Users/donmccurdy/Documents/Projects/glTF-Transform/node_modules/node-fetch/lib/index.js:1461:11)
    at ClientRequest.emit (events.js:400:28)
    at ClientRequest.emit (domain.js:475:12)
    at Socket.socketErrorListener (_http_client.js:475:9)
    at Socket.emit (events.js:400:28)
    at Socket.emit (domain.js:475:12)
    at emitErrorNT (internal/streams/destroy.js:106:8)
    at emitErrorCloseNT (internal/streams/destroy.js:74:3)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
```

Perhaps it would be easier just to 'mock' the fetch API? See how tests for WebIO do this:

https://github.com/donmccurdy/glTF-Transform/blob/6b9f4791a7c5cea1aa42674dfdf4055098a06833/packages/core/test/io/web-io.test.ts#L11-L30

In this case we wouldn't have to patch the global namespace since `fetch` is already passed as a constructor parameter.